pkgs/applications/networking/instant-messengers/telegram/cutegram/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub
+, qt5Libs, qtbase, qtquick1, qtmultimedia, qtquickcontrols, qtgraphicaleffects, makeQtWrapper
+, telegram-qml, libqtelegram-aseman-edition }:
+
+stdenv.mkDerivation rec {
+  name = "cutegram-${version}";
+  version = "v2.7.0-stable";
```

Comment:
Because version cannot start with a letter in nix. Because nix will parse name that as `cutegram-v`.